src/game/entities.cpp

```diff
@@ -1254,16 +1254,21 @@ namespace entities
         if(!m_edit(game::gamemode) && suicide) game::suicide(d, HIT_SPAWN);
     }
 
-    void editent(int i, bool local)
+
+    void editent(int i, bool send_update_to_server, bool fix_entity)
     {
         extentity &e = *ents[i];
-        if (local)
+
+        if (fix_entity)
         {
             fixentity(i, true);
-            if(m_edit(game::gamemode) && game::player1.state == CS_EDITING)
-            {
-                client::addmsg(N_EDITENT, "ri5iv", i, (int)(e.o.x*DMF), (int)(e.o.y*DMF), (int)(e.o.z*DMF), e.type, e.attrs.length(), e.attrs.length(), e.attrs.getbuf()); // FIXME
-            }
+        }
+
+        if (send_update_to_server &&
+            m_edit(game::gamemode) &&
+            game::player1.state == CS_EDITING)
+        {
+            client::addmsg(N_EDITENT, "ri5iv", i, (int)(e.o.x*DMF), (int)(e.o.y*DMF), (int)(e.o.z*DMF), e.type, e.attrs.length(), e.attrs.length(), e.attrs.getbuf()); // FIXME
```

Comment:
since #59, size() can be used for vector. Since that line is touched, why not use size() instead of length()?
Also, we're using C++, not C, take the occasion to remove the C-style casts when possible.